### PR TITLE
Add parameter to use or not the ecr-credentials-helper

### PR DESCRIPTION
**Pull request by marboledacci**
### Checklist

The credentials helper has caused many issues compared to the classic docker login, I'm adding a parameter to decide which option to use in the job and the commands.

### CODE REVIEW

File: src/commands/build_and_push_image.yml [hunk 1]
```diff
@@ -201,6 +201,12 @@ parameters:
       The alias, key ID, or full ARN of the KMS key can be specified.
     default: ""
 
+  use_credentials_helper:
+    type: boolean
+    default: true
+    description: >
+      When true the authentication to docker registry will be done using the ecr-credential-helper. This avoids having
+      the password saved in text plain. Otherwise it will use the classic docker login command which is more compatible.
```
Comment (bjohnso5) [inline src/commands/build_and_push_image.yml:209]: I'd reword this to `plain text` rather than `text plain`

Comment (marboledacci) [reply src/commands/build_and_push_image.yml]: Updated

**Review by bjohnso5 — APPROVED** [review]
